test/functional/readpreference.test.js

```diff
@@ -655,3 +656,77 @@ describe('ReadPreference', function() {
     });
   });
 });
+
+describe.only('ReadPreference - operations', () => {
+  const COLLECTION = 'ddl_collection';
+  beforeEach(function(done) {
+    const configuration = this.configuration;
+    const client = configuration.newClient(configuration.writeConcernMax(), {
+      useUnifiedTopology: true,
+      readPreference: 'primaryPreferred'
```

Comment:
why is this `primaryPreferred`?

Reply:
It just needs to not be set to `primary` and `primaryPreferred` was what the user ticket had in it.